test/framework/src/main/java/org/elasticsearch/test/AbstractBuilderTestCase.java

```diff
@@ -127,6 +127,7 @@ public abstract class AbstractBuilderTestCase extends ESTestCase {
     protected static final String GEO_POINT_ALIAS_FIELD_NAME = "mapped_geo_point_alias";
     // we don't include the binary field in the arrays below as it is not searchable
     protected static final String BINARY_FIELD_NAME = "mapped_binary";
+    protected static final String UNMAPPED_FIELD_NAME = "unmapped_field";
```

Comment:
do you need a new constant for this? After all, any field that is not listed here is unmapped in that the test base class does not add it to the mapping?